content/docs/user-guide/testing/getting-started/_index.md

```diff
@@ -0,0 +1,277 @@
+
+## Overview
+
+O3DE uses [CMake](https://cmake.org/cmake/help/latest/), a build system that includes [CTest](https://cmake.org/cmake/help/latest/manual/ctest.1.html). CTest is a generic test runner tool which coordinates executing and reporting on tests, used by O3DE to start all automated testing. O3DE projects can also register their own tests through CTest, to test their unique game code or engine extensions. By running CTest locally, developers can verify code health before committing changes.
+
+{{< important >}}
+All CTest tests are executed in the O3DE Automated Review (AR) pipeline to help prevent bad merges. New code submissions are required to have tests; Changes to existing code must not cause regressions. Make sure to run your tests locally before any code submission to the O3DE project.
+{{< /important >}}
+
+The rest of this topic covers registering tests with CTest for testing O3DE code, and how to write these tests for various test runners.
+
+![CMake and CTest workflow](/images/user-guide/testing/getting-started/cmake_ctest_workflow.png)
+
+## CTest
+
+CTest is similar to other test frameworks like GoogleTest, PyTest, JUnit, or NUnit; though it uses a higher-level generic interface. CTest registers each test as a string of command-line arguments, and reports on whether invoking those arguments succeeded (returned `0`) without hanging. CTest can also coordinate running tests in parallel processes. Unlike other test runners, CTest is programming language agnostic by interfacing with the OS shell. However this means a notable feature is absent from CTest: other test frameworks typically provide a lower-level interface which directly invokes functions inside a specific programming language. To provide this function-level execution, O3DE uses CTest to invoke lower-level test runners.
+
+O3DE provides wrapper code to help register tests for frameworks such as GoogleTest and PyTest into CTest. Along with the provided test-tools, these wrappers enable writing tests which can run on any operating system. XML files are generated to track the low-level results not reported to CTest, and artifacts are saved such as log output and crash dumps.
+
+![CTest calls other test runners](/images/user-guide/testing/getting-started/ctest_to_runners.png)
+
+In short, all tests in O3DE use CTest as a high-level test coordinator. When executed, CTest calls other lower-level test tools and then reports on their success.
+
+### Starting CTest
+
+CTest expects its working directory to be a CMake build directory, so be sure to first navigate a terminal to this directory. Then use `-C` to select a build configuration one you have already built locally:
+
+* **Windows**:
+    ```shell
+    cd <local_path_to>\\o3de\\build\\<build_folder>
+    ctest -C <build_configuration>
+    ```
```

Comment:
```suggestion
* **Windows**:
    ```shell
    cd <local_path_to>\o3de\build\<build_folder>
    ctest -C <build_configuration>
    ```
```
Wondering if it has to be double slash here? Other pages with Windows cd commands contain single slashes. E.g. [Create a new O3DE project](https://o3de.org/docs/welcome-guide/create/creating-projects-using-cli/creating-windows/#create-a-new-o3de-project)